### ISSUE TITLE
Can scDRS be used for snRNA-seq or scATAC-seq data?

### ISSUE THREAD

**Issue by adobe7105** [martinjzhang/scDRS#109]
Hi，Dr.zhang

Does scDRS work on snRNA-seq or scATAC-seq data?
scDRS 可以在 snRNA-seq 或 scATAC-seq 数据上工作吗？

以下是我的理解， snRNA-seq可以，但scATAC-seq不行。
Here is my understanding, scDRS can run on snRNA-seq but not on scATAC-seq.

scRNA-seq与snRNA-seq的区别在于所检测的对象是否只为细胞核。简单说，sn包含了更多，但nucleus细胞核所包含的RNA，但因为sn包含了尚未被切割的RNA intron，sc只包含了成熟的mRNA，所以sn可以同時取得exon与intron的讯息。而scpagwas利用与gwas高度关联的基因作为识别细胞类型的依据，这在snRNA-seq中是符合的
The difference between scRNA-seq and snRNA-seq depends on whether the detected object is only the nucleus. Simply put, sc contains more RNA than the nucleus contains, but because sn contains uncut RNA intron, sc only contains mature mRNA, so sn can obtain information from exon and intron at the same time.
scDRS uses genes highly related to gwas as the basis for identifying cell types, which is consistent with snRNA-seq
snATAC-seq则是基于转座酶的染色质可及性测序技术，利用转座酶可以随机转座到细胞核内染色质的开放区域，并加上接头，扩增建库，获得整个基因组内染色质开放区域的DNA序列（只包括染色质区域）
snATAC-seq is a chromatin accessibility sequencing technology based on transposase. It uses transposase to randomly transpose to the open area of ​​chromatin in the cell nucleus, add adapters, amplify and build a library, and obtain the DNA sequence of the open area of ​​chromatin in the entire genome (only including the chromatin area)

Am I right？

yours,zemin

by.another question is ,Can scDRS be used for gwas with quantitative traits as outcomes? like aging？
另一个问题是，scpagwas可以被用于被非二元结局的gwas吗？比如年龄、等衰老指标

**Comment by martinjzhang** [comment]
yes, scDRS can be used for  snRNA-seq data. We are currently developing a new method for scATAC-seq, involving different normalization procedures. 

>by.another question is ,Can scDRS be used for gwas with quantitative traits as outcomes? like aging？
It should be fine.
